frontend/components/AddItemForm.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import { View, StyleSheet, ScrollView, KeyboardAvoidingView } from 'react-native';
+import { View, StyleSheet, ScrollView, KeyboardAvoidingView, Image } from 'react-native';
 import { Ionicons } from '@expo/vector-icons';
 import { Input, Text, Button } from 'react-native-elements';
 import { addGroceryItem } from '../utils/api';
+import { ImageManipulator } from 'expo';
 
```

Comment:
This seems to be one of your longer files and so you should consider trying to condense it (see other suggestion about moving stylesheet for example.

Reply:
Hmmm moving the styles is a good suggestion but it would only remove 15 lines. Perhaps I can add some comments to make the code more readable